### PR TITLE
Doc for new certificate configuration

### PR DESCRIPTION
**Pull request by vit-tomica**
#### Is there a related issue for this PR? 
Issue number: https://github.com/zowe/zowe-install-packaging/issues/741

### CODE REVIEW

**Comment by nannanli** [comment]
@vit-tomica @stevenhorsman Hi Vit, Steve, 
I have one question regarding this file. It seems some content in this file is duplicate of the content in this section on the doc site: https://docs.zowe.org/stable/user-guide/configure-zowe-runtime.html#configuration-variables. Does this file replace the cited section? 

Also I'm wondering when should a user configure the certificates, specifically, after which configuration step should this configure certificates step be performed? Just trying to understand where to add this file to the navigation. Thank you!

**Comment by stevenhorsman** [comment]
@nannanli Hey Ashley, this is part of the new Cupids phase 1 install and configuration process. The rest of the documentation will need to be updated to match as things get completed, but the overview is: the current yaml configuration is going and instead there we be the following stages:
1. Install the product
2. Limited time only configuration: running the ZWESECUR set-up to create the security configuration, creating the keystore using these instructions and doing the initial configure of the cross memory server similarly to the details in the current configure runtime section
3. Create a Zowe instance - this sets up a new directory which has a customisable config file (including a link to the keystore directory) and stores the user runtime information.

In order to not disrupt the user experience the changes (including keystore customisation) have been made whilst being backwardly compatible, but this is due to change soon as we switch over and this file will be part of the new process documentation. I fully expect the CUPIDS squad to create the above framework in the doc, but until then I don't think we need to link to this new doc. I hope this helps provide a bit more context.

File: docs/user-guide/configure-certificates.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+Following is the part of `zowe-setup-certificates.env` file snippet that uses existing certificates:
+```shell script
+# Should APIML verify certificates of services - true/false
+VERIFY_CERTIFICATES=true
+# optional - Path to a PKCS12 keystore with a server certificate for APIML
+EXTERNAL_CERTIFICATE=/path/to/keystore.p12
+# optional - Alias of the certificate in the keystore
+EXTERNAL_CERTIFICATE_ALIAS=servercert
+# optional - Public certificates of trusted CAs
+EXTERNAL_CERTIFICATE_AUTHORITIES=”/path/to/cacert_1.cer /path/to/cacert_2.cer”
```
Comment (plavjanik) [inline docs/user-guide/configure-certificates.md]:
```suggestion
EXTERNAL_CERTIFICATE_AUTHORITIES="/path/to/cacert_1.cer /path/to/cacert_2.cer"
```

**Comment by nannanli** [comment]
@stevenhorsman Thank you Steve! That clarifies my confusion and helps a lot to understand where we're going.

**Comment by vit-tomica** [comment]
@stevenhorsman Steve, can you please give me write access to this repo (zowe/docs-site) so I can properly sing off a commit?

**Comment by stevenhorsman** [comment]
@vit-tomica - Vitek, you should have permissions now. Let me know if it didn't work

**Comment by MarkAckert** [comment]
All checks passed besides build, and the build failure is from the upstream branch. Merging.

**Pull request merged**